src/components/authorization/roles/consultant-on-project.role.ts

```diff
@@ -335,31 +335,31 @@ export const ConsultantOnProject = new DbRole({
       __className: 'DbProject',
       canList: true,
       properties: [
-        { propertyName: 'estimatedSubmission', permission: { read, write, }, },
-        { propertyName: 'step', permission: { read, write, }, },
-        { propertyName: 'name', permission: { read, write, }, },
-        { propertyName: 'status', permission: { read, write, }, },
-        { propertyName: 'departmentId', permission: { read, write, }, },
-        { propertyName: 'mouStart', permission: { read, write, }, },
-        { propertyName: 'mouEnd', permission: { read, write, }, },
-        { propertyName: 'initialMouEnd', permission: { read, write, }, },
-        { propertyName: 'stepChangedAt', permission: { read, write, }, },
-        { propertyName: 'rootDirectory', permission: { read, }, },
-        { propertyName: 'member', permission: { read, write, }, },
-        { propertyName: 'otherLocations', permission: { read, write, }, },
-        { propertyName: 'primaryLocation', permission: { read, write, }, },
-        { propertyName: 'marketingLocation', permission: { read, write, }, },
-        { propertyName: 'partnership', permission: { read, write, }, },
-        { propertyName: 'budget', permission: { read, write, }, },
-        { propertyName: 'modifiedAt', permission: { read, write, }, },
-        { propertyName: 'fieldRegion', permission: { read, write, }, },
-        { propertyName: 'engagement', permission: { read, write, }, },
-        { propertyName: 'sensitivity', permission: { read, write, }, },
+        { propertyName: 'estimatedSubmission', permission: { read, }, },
+        { propertyName: 'step', permission: { read, }, },
+        { propertyName: 'name', permission: { read, }, },
+        { propertyName: 'status', permission: { read, }, },
+        { propertyName: 'departmentId', permission: { }, },
+        { propertyName: 'mouStart', permission: { read, }, },
+        { propertyName: 'mouEnd', permission: { read, }, },
+        { propertyName: 'initialMouEnd', permission: { read, }, },
+        { propertyName: 'stepChangedAt', permission: { read, }, },
+        { propertyName: 'rootDirectory', permission: { read, write }, },
+        { propertyName: 'member', permission: { read, }, },
+        { propertyName: 'otherLocations', permission: { read, }, },
+        { propertyName: 'primaryLocation', permission: { read, }, },
+        { propertyName: 'marketingLocation', permission: { }, },
+        { propertyName: 'partnership', permission: { read, }, },
+        { propertyName: 'budget', permission: { }, },
+        { propertyName: 'modifiedAt', permission: { read, }, },
+        { propertyName: 'fieldRegion', permission: { read, }, },
+        { propertyName: 'engagement', permission: { read, }, },
+        { propertyName: 'sensitivity', permission: { read, }, },
         { propertyName: 'tags', permission: { read, write, }, },
```

Comment:
Just double-checked with Carson and we can go ahead and delete write for `tags`, too

Reply:
Ok I'm on it